telegram/constants.py

```diff
@@ -753,6 +806,25 @@ class MessageType(StringEnum):
     """:obj:`str`: Messages with :attr:`telegram.Message.video_chat_participants_invited`."""
 
 
+class NumberOfUpdatesLimit(IntEnum):
```

Comment:
spontanious naming idea: `PollingLimits`? that way this could include any limits that are related to (long) polling, specifically `get_updates`

Reply:
Done in fcf6e57164278bf2c2381d42334a104485e81b08 in #3351